packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -33,14 +43,14 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScr
     super(props);
     this._subscribers = new Set<Function>();
     this._stickies = new Set<Sticky>();
-
+    const { scrollbarVisibility } = this.props;
     this.state = {
       stickyTopHeight: 0,
       stickyBottomHeight: 0,
-      scrollbarWidth: 0,
-      scrollbarHeight: 0
+      scrollbarWidth: scrollbarVisibility === ScrollbarVisibility.always ? this._getScrollbarHeightFromLocalStorage(true) : 0,
+      scrollbarHeight: scrollbarVisibility === ScrollbarVisibility.always ? this._getScrollbarHeightFromLocalStorage(true) : 0
```

Comment:
Nice! I hope we were able to save a re-render due to this.

Reply:
Using `window.localStorage` for scroll position seems like an app-level concern rather than a scrollable DIV's.

Reply:
It's not scroll position but scrollbar height/width which is getting stored in local storage. [StickyAboveContainer](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx#L180
) needs vertical scrollbar's width and [StickyBelowContainer ](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx#L181
)needs horizontal scrollbar's height. The height of horizontal scrollbar would always be same as the width of vertical scrollbar, that's why only height is being calculated/stored.